### ISSUE TITLE
Mindslave Implant is broken as shit and we all know it.

### ISSUE THREAD

**Issue by ParagonPrime** [yogstation13/yogstation-classic#284]
It's the new combo cafe, except only a few people get to be in on it!

Mindslave implant itself is fine. No problems there. The only problem is that it's become nothing more than a huge clusterfuck of metafriending.

Someone pointed out on forums that metafriends would just wait for one of them to be antag, then poke the other with mindslave and go around all merry.  Well, unfortunately, this is very much what has happened- almost every single fucking round, doubly so on lowpop, it's the same people going around doing the antag work, and in the same pairs. You know who these people are just as well as I do.

People are gonna bitch about me suggesting this. People are gonna call me salty. People are gonna tell me to git gud. But there's an undeniable truth: _mindslave implant is little but a legitimized form of metafriending, and there is no way to fix that short of removing it._

**Comment by oisin100** [comment]
I agree its pretty shitty to other players when someone uses the implant to metafriend...

I have a couple of options what might help:
1. Perhaps we could make it so the admins can get a full log of the interactions between the slave and the master so they can tell how much they actually communicated with a simple click of 1 button instead of having to dive neck deep in logs
2. The implant will auto-detonate after 45 Minutes or so.

**Comment by ParagonPrime** [comment]
1. Sounds good because you can actually catch metafrienders rather than force them to hide in the shadows.
2. Doesn't deal with the problem, really. Overall a very blunt solution.

**Comment by X-TheDark** [comment]
There is absolutely no way to deal with this short of rules (and therefore admin intervention) or extremely invasive mechanics, because the whole premise is to implant someone who is useful to you, either through their job, their in-game name/reputation or both.

My idea of the more draconian and heavy-handed solution is to simply re-roll the person, i.e., when someone gets implanted, similarly to normal antag selection from pool of ghosts that have the antag preference enabled. Yes, this means the implanted person will probably not be playing the antag, but damn, as Paragon said, I see no way to relieve this without drastic measures or outright removal, as, indeed, this is nothing more than legal metafriending.

**Comment by ShadowDeath6** [comment]
I don't really understand why this is a problem. I mean, yeah, it lets people antag their friends, but the point of mindslave implant is to get someone who you know will help you onto your side. If admins start noticing a trend of the same two people implanting each other over and over, it's probably up to them to do something about it.

**Comment by ParagonPrime** [comment]
Problem is, Shadow, this always tends to happen on the skeleton crew rounds with _no admins on_, so... they can't really do anything about it as such!

Think of Mindslave in the context of non-meta play.

You're meant to abduct and jab with Mindslave whoever it is that'll be most useful to helping you complete your objective: HoP if you need Cap's laser, CE if you need to fuck up tcomms, etcetera etcetera. Instead, it becomes a case of figuring out who your friend on the server is and jabbing them- even if you need to steal the Ablative and the person you mindslave happens to be a fucking Paramedic or something.

This could go even beyond that, though: people deliberately picking certain jobs because they know their buddy is gonna get antag and mindslave them, so now they've put themselves in optimum position to be an antag.

TL;DR - Mindslave is a nice concept and would be fucking amazing if Yogs wasn't infested with powergamers who take advantage of it in a metagaming way.

**Comment by ghost** [comment]
Mindslave IS to be used for metafriend ect. that's the whole point.

You're supposed to implant people you know are competent and you know will help you and like you.

There's no fucking 'equal muh opportunities' implants. You aren't going to implant baldface mcgee over Phazon the mime because the assistant will probably just toolbox you and bekons is robust.

If it happens tho that they know X got antag and they work to get together and work together it's an admin issue. And by an admin issue I mean I ban them both for metacomms.

And aside from that it's easy to note if someone's going around mindslaving the same people every round. and again, it'll be netted with a ban just the same.

You can't avoid an admin noticing you. Every part of this game depends on admin intervention to keep it from being a grief fest-If you see someone metafriending implants, AHELP IT, and we'll take care of it.

Non-issue.

**Comment by FluffySurvivor** [comment]
@AdamElTablawy 
But what if bekons play several rounds in a row and mister antag gets antag in a row as well ? he'll still be the same option for implanting each round
About the item, it really has no downsides.
Robust player active in the game ? implant easy greentext
No robust player active ? dont buy implant buy something else in the traitor meta

**Comment by ghost** [comment]
no traitor items have downsides, fluffy

if they had downsides to buying them they wouldn't work

hacked ai boards and multitools are only useful if there's an ai, if there's not it's a waste of tc

freedom implant is only useful if you're going to get caught by sec

ect

**Comment by FluffySurvivor** [comment]
except the esword that everyone buys because it's cool even if it underperforms

ebow is expensive af and is in most cases a noob bait since you can just snag a taser 

hacked ai module is meta-dependant just like the mindslave, most sillicons nowadays are stupid

**Comment by Xantam-13** [comment]
As someone suggested earlier, I think the best way to do it is essentially a sacrifice. You implant someone you don't give a shit about loosing. 

For example, I implant Asv9 to do my bidding. I know in my mind the implant triggers a "timebomb in the player" which I'd actually set to like 20 minutes. So the slave has 20 minutes of slavery before the player is gibbed. If you implanted a friend, you're essentially killing your friend. 45 minutes is way too long. The implant is just replacing "parapen, sword" and should be a little more friendly. 

I'd go with;
Implant
Instagib after 20 minutes of slavery. This also means they have to move faster to accomplish their objectives. 
Perfect solution in my opinion. I'd actually be more inclined to implant my enemy in this fashion.

**Comment by oisin100** [comment]
Alright then. But the question we need to ask is would you use the implant if it only lasted a certain amount of time?

I remember there being something like the mindslave implant before but it was never used.

**Comment by X-TheDark** [comment]
Perhaps if you made it cheaper to compensate. Like 7-8 TC, since you will lose your "investment" in 20 minutes. Sure, it gives you 2 slaves, but you need to act fast and furious. This would basically be Nuke Op LOUD mode.

**Comment by oisin100** [comment]
Alright, Changing it to 8TC per implant and making them explode after 20 minutes.

**Comment by ParagonPrime** [comment]
Yep, that sounds much more fair.

**Comment by ghost** [comment]
Instead of explode, why not just decay?

As in, the implant wears off after 20-30 minutes, and whoever was implanted remembers everything that happened and their master's name.

So that the master would actually have to finish it off with ordering their mindslave to kill themselves before time ran out.

**Comment by FluffySurvivor** [comment]
Self destructing mindaves would make the item chameleon projector tier. Just don't ruin it. 
Seriously, a robust player mindslaves another robust player in low pop against unrobust af security/crew with no high power tier non antags players to counter them and everyone loses their minds. A lot of shit is unbalanced at low pop, and mindslave just enters the category. Once there is numerous competent opposition, you'll regret the fucking 12 tc wasted because you have no offensive support. If you feel like nerfing it, make it cost 13 TC and bam you can't afford an esword or adrenals, which basically makes your antag round dependent on teamwork because you have no offensive nor defensive options.

**Comment by Xantam-13** [comment]
The only reason I suggested/supported death of the implanted victim is to stop metafriending. No one will knowingly implant their friend knowing it'd kill them permanently in 20 minutes. That is the essence of my suggestion.

**Comment by FluffySurvivor** [comment]
I don't see an issue implanting a "friend". Unless they are metacomming, that is the point of the implant. You spend more than half of your TC pool to gain an powerful ally that will help you and make both players have fun. You are not giving the buddy god mode. It's much more fun to implant someone who are a pleasure to cooperate with.

**Comment by Stealthkibbler** [comment]
In my experience of utilizing the implant why not just use it as a way to change revolution? People wanted to make revolution different, we could effectively just remove it from the traitor uplink and/or make it the recruitment item for revolution heads, this would require revolution heads to be less bumrush as most people have complained about while at the same time offering a different and fresh experience to the revolution gamemodes meta.

**Comment by ghost** [comment]
> Stealthkibbler commented 13 hours ago
> In my experience of utilizing the implant why not just use it as a way to change revolution? People wanted to make revolution different, we could effectively just remove it from the traitor uplink and/or make it the recruitment item for revolution heads, this would require revolution heads to be less bumrush as most people have complained about while at the same time offering a different and fresh experience to the revolution gamemodes meta.

THIS THIS THIS THIS

**Comment by Alblaka** [comment]
Remove metafriending implant when :(

**Comment by ghost** [comment]
Close this. It is done.

**Comment by oisin100** [comment]
As adam said.
